runtime/mangata-kusama/src/lib.rs

```diff
@@ -1098,6 +1098,78 @@ impl orml_asset_registry::Config for Runtime {
 	type WeightInfo = weights::orml_asset_registry_weights::ModuleWeight<Runtime>;
 }
 
+// Proxy Pallet
+/// The type used to represent the kinds of proxying allowed.
+#[derive(
+	Copy,
+	Clone,
+	Eq,
+	PartialEq,
+	Ord,
+	PartialOrd,
+	Encode,
+	Decode,
+	RuntimeDebug,
+	MaxEncodedLen,
+	TypeInfo,
+)]
+pub enum ProxyType {
```

Comment:
when proxy call is made, whoe defines the ProxyType, is it caller?

Reply:
yes, you as a user/caller select the ProxyType/access level to your account

Reply:
@mateuszaaa see integration tests for proxy